src/MultiAgentReinforcementLearning/src/policies/multi_agent.jl

```diff
@@ -0,0 +1,56 @@
+export MultiAgentManager
+
+Base.@kwdef mutable struct MultiAgentManager{T <: AbstractPolicy} <: AbstractPolicy
+    agents::Dict{S,T} where {S<: Union{String, Symbol, Integer}}
+    cur_player::Union{String, Symbol, Integer} = first(keys(agents))
+end
+
+Base.getindex(A::MultiAgentManager, x) = getindex(A.agents, x)
+
+"""
+    MultiAgentManager(player => policy...)
+
+This is the simplest form of multiagent system. At each step they observe the
+environment from their own perspective and get updated independently.
+"""
+MultiAgentManager(policies...) =
+    MultiAgentManager(Dict{Any,Any}(nameof(p) => p for p in policies))
```

Comment:
Can we avoid allocating a `Dict{Any,Any}`? Can we get away with a `NamedTuple`?

Reply:
Actually, this part stayed as it was done by @peterchen96. In my opinion, we do not need it anyway. `NamedPolicy` is no part of the module as well.

Reply:
@jeremiahpslewis do you think that it this constructor should be present for ease of use?

Reply:
```julia
Base.@kwdef mutable struct MultiAgentManager{T <: AbstractPolicy, S} <: AbstractPolicy
    agents::Dict{S,T}
    cur_player::S = first(keys(agents))
end
``` 

Base.@kwdef automatically defines the constructor. Hence, this part is not necessary in my opinion.